adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/DefaultDeviceResolver.java

```diff
@@ -193,9 +192,9 @@ private void loadCredentialsForDevice(final ConnectionId cid, final PskPublicInf
         final String publicInfo = identity.getPublicInfoAsString();
         LOG.debug("getting PSK secret for identity [{}]", publicInfo);
 
-        final Span span = newSpan("PSK-getSecretKey");
+        final Span span = newSpan("PSK-getDeviceSecretKeyAndIdentityInfo");
```

Comment:
I would prefer: "PSK-getDeviceCredentials"

Reply:
Good, I've changed it accordingly.